Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM node:11.15-stretch
+FROM node:16.15-stretch
```

Comment:
This image is no longer supported (it's up to `16.16` already and `stretch` is too old a version of Debian), could we use `node:16-bullseye` instead? That way we don't have to update the `Dockerfile` every time a new point release comes out.

Alternatively `node:16.16-bullseye`, `node:16-buster`, or `node:16.16-buster` would work fine too.

Reply:
Yes!  See #1077

Reply:
Great, thanks!